src/averageAges.js

```diff
@@ -20,6 +20,20 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const men = people.filter(
+    century !== undefined
+      ? person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century
+      : person => person.sex === 'm'
+  );
```

Comment:
looks like you duplicate yourself with `person.sex === '...'` not only here but all over your code
I'd rather create a function filtering people by sex so that it looks simpler
```
const men = ...;
const menDiedInCentury = men.filter(...);
```